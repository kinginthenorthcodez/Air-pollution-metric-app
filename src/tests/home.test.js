import { render, cleanup } from '@testing-library/react';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom/extend-expect';
import store from '../redux/configureStore';
import Home from '../components/home';

// Home component tests
describe('Home snapshot', () => {
  afterEach(cleanup);

  it('should render Home correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    expect(asFragment(<Home />)).toMatchSnapshot();
  });
});
