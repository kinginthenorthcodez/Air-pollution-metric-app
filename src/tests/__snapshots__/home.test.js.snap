// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot should render Home correctly 1`] = `
<DocumentFragment>
  <section
    class="region-container"
  >
    <div
      class="item"
    >
      <div
        class="header"
      >
        <h2>
          Map region
        </h2>
        <div
          class="title"
        >
          <h1>
            Africa
          </h1>
        </div>
      </div>
      <input
        class="input"
        placeholder="Search your favorite region..."
        type="text"
        value="Africa"
      />
      <div
        class="section"
      />
    </div>
  </section>
</DocumentFragment>
`;
